routers/api/v1/api.go

```diff
@@ -771,7 +771,7 @@ func Routes() *web.Route {
 				m.Get("/attachment", settings.GetGeneralAttachmentSettings)
 				m.Get("/repository", settings.GetGeneralRepoSettings)
 			})
-		}, tokenRequiresScopes(auth_model.AccessTokenScopeCategoryMisc))
```

Comment:
Then `AccessTokenScopeCategoryMisc` will not be used in any other place, should we also remove it from models/auth/token_scope.go ?

Reply:
Added a comment here.